legal-api/src/legal_api/reports/business_document.py

```diff
@@ -122,6 +122,7 @@ def _get_template_data(self, get_json=False):
             # get document data
             business_json['reportType'] = self._document_key
             business_json['business'] = self._business.json()
+            business_json['business']['legalName'] = self._business.legal_name  # legal name easy fix
```

Comment:
Does this required? business model handled it (above ⬆️)

Reply:
Yes. Outputs should use the old legal name. That is, it should be operating name for firms. But json now returns the correct legal name, which is not the operating name.